packages/react-button/src/components/MenuButton/MenuButton.types.ts

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { ShorthandProps, ObjectShorthandProps } from '@fluentui/react-utilities';
+import { ButtonProps, ButtonState, ButtonStyleSelectors, ButtonTokens, ButtonVariants } from '../Button/Button.types';
+
+/**
+ * {@docCategory Button}
+ */
+export type MenuButtonProps = Omit<ButtonProps, 'iconPosition'> & {
+  /**
+   * Menu icon that indicates that this button has a menu that can be expanded.
+   */
+  menuIcon?: ShorthandProps<React.HTMLAttributes<HTMLElement>>;
+
+  /**
+   * Defines the initial expanded state of the MenuButton.
+   * Use this if you want the MenuButton to maintain its own state.
+   * Mutually exclusive with `expanded`.
+   * @default false
+   */
+  defaultExpanded?: boolean;
+
+  /**
+   * Defines whether the MenuButton is in an expanded state.
+   * Use this if you wish to have the expanded state of the MenuButton be controlled.
+   * Mutually exclusive with `defaultExpanded`.
+   * @default defaultExpanded
+   */
+  expanded?: boolean;
```

Comment:
Again, this state of `MenuButton` will be duplicating `Menu` state

Reply:
Commented until we have a good contract with `Menu`.